### PR TITLE
feat: add mongoatlas cost client

### PR DESCRIPTION
**Pull request by holiiveira**
Add cloud provider MongoAtlas.

Add the following configurations to your `app-config.yaml`:

```diff
backend:
+  infraWallet:
+    integrations:
+      mongoatlas:
+        - name: 
+          orgId: 
+          publicKey: 
+          privateKey: 
```

TODO:

- [x] Add documentation
- [x] Improve category mappings
- [x] Add mock

fix: #57

### CODE REVIEW

**Comment by holiiveira** [comment]
Hi @gluckzhang, I thought it might be a set of fixed meta information, but it's not, our CSVs:

```csv
Invoice Number,xxxx,
Billing Period,"July 1, 2024 - August 1, 2024",
Organization Name,xxxx,
Organization ID,xxxx,
```

> In the implementation, instead of slicing a fixed number of lines, what about skipping lines until this Organization ID is reached?

I made some changes, please check.

File: plugins/infrawallet-backend/seeds/default_category_mappings.js [hunk 1]
```diff
@@ -476,5 +476,20 @@ exports.seed = async knex => {
         'Service Infrastructure',
       ]),
     },
+    {
+      provider: 'mongoatlas',
+      category: 'Storage',
+      cloud_service_names: JSON.stringify(['Atlas Cloud Backup Storage - Azure', 'Atlas Storage 32 GB - Azure']),
```
Comment (gluckzhang) [inline plugins/infrawallet-backend/seeds/default_category_mappings.js:482]: If you have pulled the latest code from `main` branch, you will see that now category mapping supports regex. So I would suggest something like `['Atlas Cloud Backup Storage', 'Atlas Storage']` for this category. In our case, we have `Atlas Cloud Backup Storage - AWS` in SKU but we don't need to list all of them here :)

File: plugins/infrawallet-backend/seeds/default_category_mappings.js [hunk 2]
```diff
@@ -476,5 +476,20 @@ exports.seed = async knex => {
...
+    },
+    {
+      provider: 'mongoatlas',
+      category: 'Networking',
+      cloud_service_names: JSON.stringify(['Atlas Azure Data Transfer']),
```
Comment (gluckzhang) [inline plugins/infrawallet-backend/seeds/default_category_mappings.js:487]: `['Data Transfer']` should work well enough.

File: plugins/infrawallet-backend/seeds/default_category_mappings.js [hunk 3]
```diff
@@ -476,5 +476,20 @@ exports.seed = async knex => {
...
+    },
+    {
+      provider: 'mongoatlas',
+      category: 'Compute',
+      cloud_service_names: JSON.stringify(['Atlas M10 Instance - Azure', 'Atlas M20 Instance - Azure']),
```
Comment (gluckzhang) [inline plugins/infrawallet-backend/seeds/default_category_mappings.js:492]: Let's try `['Instance']` here, which covers all SKUs like `Atlas XX Instance - YY`. Not sure if some other SKUs will contain the `Instance` key word, but we can use it for now.

**Review by gluckzhang — APPROVED** [review]

**Comment by gluckzhang** [comment]
Reviewed and tested in our environment, it works very well. Thank you @holiiveira again for the PR!

**Pull request merged**